### PR TITLE
Update protobuf to 2.0.2

### PR DESCRIPTION
**Pull request by driftluo**
Update protobuf to 2.0.2

### CODE REVIEW

File: libproto/Cargo.toml [hunk 1]
```diff
@@ -5,7 +5,7 @@ version = "0.6.0"
 authors = ["Cryptape Technologies <contact@cryptape.com>"]
 
 [dependencies]
-protobuf = { version = "^1.0.0"}
+protobuf = { version = "^2.0.2", features = ["with-bytes"]}
```
Comment (chaoticlonghair) [inline libproto/Cargo.toml]: Remove `^` ?

File: libproto/create_protobuf.sh [hunk 2]
```diff
@@ -174,9 +174,9 @@ function main () {
     check_dependencies
     remove_all_rs
     gen_rs_for_protos
-    add_pub_to_oneof_in_generated_code ./src/protos/response.rs      data    Response
-    add_pub_to_oneof_in_generated_code ./src/protos/request.rs       req     Request
-    add_pub_to_oneof_in_generated_code ./src/protos/communication.rs content InnerMessage
+#    add_pub_to_oneof_in_generated_code ./src/protos/response.rs      data    Response
```
Comment (chaoticlonghair) [inline libproto/create_protobuf.sh]: Delete all useless codes, include the `function` if it is useless. We have [VCS](https://en.wikipedia.org/wiki/Version_control).

**Review by u2 — DISMISSED** [review]

**Comment by u2** [comment]
CI

**Review by chaoticlonghair — DISMISSED** [review]

**Comment by chaoticlonghair** [comment]
Keep patient!
Block by [PR-89: Upgrade the build environment](https://github.com/cryptape/cita-common/pull/89).

**Review by TheWaWaR — DISMISSED** [review]

**Review by chaoticlonghair — APPROVED** [review]

**Review by u2 — APPROVED** [review]

**Pull request merged**